src/components/Documents/DocumentTableRow.jsx

```diff
@@ -6,6 +6,7 @@ import { useSession } from '@hooks';
 import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
 import FileOpenIcon from '@mui/icons-material/FileOpen';
 import IconButton from '@mui/material/IconButton';
+import ShieldIcon from '@mui/icons-material/Shield';
```

Comment:
I think we could leave the permission icons with the settings icon/gear.

Personally, I think that's more intuitive than a shield.